interface/wx/ribbon/art.h

```diff
@@ -8,7 +8,7 @@
 /**
     Identifiers for common settings on ribbon art providers which can be used
     to tweak the appearance of the art provider.
-    
+
```

Comment:
It would be great to separate all whitespace-only changes in a separate commit (or avoid them entirely).